src/hello.js

```diff
@@ -63,10 +63,10 @@ client.once('READY', () => {
      * Evaluate flag using attribute
      */
     const variationWhenAttribute = client.variation(
-        'flag-1',
+        FEATURE_FLAG_KEY,
         'user-id-123',
         {
-            "age": "12",
+            "age": 12,
             "country": "US"
         });
```

Comment:
We are still connecting to qa-unlaunch, not prod, because we haven't released the change that is in develop now. I tried to test with number, but it seems to return incorrect result. We can set up a session and debug together.